HousingSearchApi/V1/Boundary/Requests/GetAccountListRequest.cs

```diff
@@ -0,0 +1,23 @@
+namespace HousingSearchApi.V1.Boundary.Requests
+{
+    public class GetAccountListRequest : HousingSearchRequest
+    {
+        /*private const int DefaultPageSize = 12;
```

Comment:
should this be commented out?

Reply:
in the next pr i'm going to make the HousingSearchRequest abstract class for follow the inheritance instruction.